src/pybind/mgr/orchestrator.py

```diff
@@ -716,6 +718,69 @@ def __init__(self):
         # some replicaset special sauce for autoscaling?
         self.extended = {}
 
+        # Object with specific settings for the service
+        self.service_spec = None
+
+class RGWSpec(object):
+    """
+    Settings to configure a multisite Ceph RGW
+    """
+    def __init__(self, hosts=None, rgw_multisite=True, rgw_zone="Default_Zone",
+              rgw_zonemaster=True, rgw_zonesecondary=False,
+              rgw_multisite_proto="http", rgw_frontend_port="8080",
+              rgw_zonegroup="Main", rgw_zone_user="zone.user",
+              rgw_realm="RGW_Realm", system_access_key=None,
+              system_secret_key=None):
+
+        self.hosts = hosts
+        self.rgw_multisite=rgw_multisite
```

Comment:
Cosmetic (aren't you using pyling for catching these things)?
```suggestion
        self.rgw_multisite = rgw_multisite
```

Reply:
VsCode does not detect this .... investigating what  is wrong with my env... :-(

Reply:
I'm sure there's some pylint plugin for vscode, but alternatively you may want to add a pre-commit hook to your git repo, and ensure tox runs pylint (an example of how we do this for dahsboard: https://github.com/rhcs-dashboard/ceph-dev/blob/master/docker/ceph/ci)